tests/tests.cmake

```diff
@@ -170,11 +230,285 @@ function(add_xc7_test)
                     --fn_in ${fasm}
                     --bit_out ${bit}
             DEPENDS
-                ${fasm}
+                xc7-${test_name}-fasm
             )
 
         add_custom_target(xc7-${test_name}-bit DEPENDS ${bit})
         add_dependencies(all-xc7-tests xc7-${test_name}-bit)
     endforeach()
 
 endfunction()
+
+function(add_xc7_validation_test)
+    # ~~~
+    # add_xc7_validation_test(
+    #    name <name>
+    #    board_list <board>
+    #    [enable_vivado_test]
+    #    [testbench]
+    # )
+    #
+    # Generates targets to run desired tests
+    #
+    # Arguments:
```

Comment:
The enable_vivado_test and testbench arguments are missing from this list

Reply:
Added missing args (fixed also from the add_xc7_tests)